### ISSUE TITLE
Can't train the object detection network with KITTI

### ISSUE THREAD

**Issue by howtocodewang** [NVIDIA/DIGITS#1330]
When I have finished created the data set for KITTI, I planed to train the network as the introduction in [https://github.com/NVIDIA/DIGITS/tree/digits-5.0/examples/object-detection](url).

However I encounter a error which is `ERROR: Check failed: img_height == data_mean_.height() (0 vs. 384)`.

 What does this error mean? And What should I do? Thanks for any help!!!

**Comment by DarylWM** [comment]
Is "Subtract Mean" set to None?

**Comment by howtocodewang** [comment]
I'm sorry to reply so late. "Subtract Mean" didn't set to None. Let me try again and check if it will work or not. Thank you very much.

**Comment by howtocodewang** [comment]
@DarylWM Thanks for your reply and I have not checked notifications till today. And you are right, I didn't set "Subtract Mean" to None. I will change this mistake and try again. Thank you!
